xmppserver/src/main/java/org/jivesoftware/openfire/handler/PresenceUpdateHandler.java

```diff
@@ -161,7 +149,7 @@ private void process(Presence presence, ClientSession session) throws Unauthoriz
             Presence.Type type = presence.getType();
             // Available
             if (type == null) {
-                if (session != null && session.getStatus() == Session.STATUS_CLOSED) {
+                if (session != null && session.getStatus() == Session.Status.CLOSED) {
```

Comment:
There already exists such a method, in `org.jivesoftware.openfire.session.Session#isClosed`:

    /**
     * Returns true if the connection/session is closed.
     *
     * @return true if the connection is closed.
     */
    default boolean isClosed() {
        return getStatus() == Status.CLOSED;
    };

The reason that I was not using it, is because some implementations override the definition (and not use the 'status' value to determine if the session is closed). I'm not sure if this is more, or less reason to start using the `isClosed()` method instead of comparing the `status` value directly. For now, I choose to not change behavior much. There probably is an argument to be made to do that, but perhaps that warrants it's own issue number?